src/cargo/core/global_cache_tracker.rs

```diff
@@ -863,9 +857,6 @@ impl GlobalCacheTracker {
         child_base_paths: &[&Path],
         delete_paths: &mut Vec<PathBuf>,
     ) -> CargoResult<()> {
-        let _p = crate::util::profile::start(format!(
```

Comment:
`parent_table_name` is missing in instrument, and ditto in global_cache_tracker.rs

Reply:
See #13538